File: InMobi/src/main/java/com/mopub/mobileads/InMobiGDPR.java
package com.mopub.mobileads;

import androidx.annotation.Nullable;

import org.json.JSONObject;

public class InMobiGDPR {
    private static JSONObject consent = null;

    /**
     * Call InMobiGDPR.setGDPRConsentDictionary() to provide GDPR consent to InMobi.
     */
    public static void setGDPRConsentDictionary(@Nullable JSONObject consentDictionary) {
        consent = consentDictionary;
    }

    /**
     * Call InMobiGDPR.getGDPRConsentDictionary() to get the last updated consent.
     */
    @Nullable
    public static JSONObject getGDPRConsentDictionary() {
        return consent;
    }
}

File: InMobi/src/main/java/com/mopub/mobileads/InMobiGDPR.kt
package com.mopub.mobileads

import com.inmobi.sdk.InMobiSdk
import org.json.JSONObject

class InMobiGDPR {
    companion object {
        @JvmStatic
        fun setGDPRConsentDictionary(consentDictionary: JSONObject?) {
            InMobiSdk.updateGDPRConsent(consentDictionary)
        }
    }
}